_posts/2019-03-20-Authentication-as-a-Service.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: "Authentication-as-a-Service"
+tags: OAuth2
```

Comment:
Check for existing tags here: https://montrehack.ca/archives.html and try to fit into one so we avoid exploding the number.

I would put `web` and `pentest` (because it's a multi-vuln setup that mimics a pentest)